fix: remove noise from `docker-compose` on build and reformat `ddev start` output, for #6905

stasadev opened the pull request:
## The Issue

@rpkoller pointed out that we have some strange output on `ddev start`:

```
Building project images... 
. db  Built 
 web  Built 
Project images built in 1s.
```

Previously it looked like this:

```
Building project images... 
.Project images built in 1s.
```

This noise was added with:

- #6905

## How This PR Solves The Issue

1. Removes the noise.
2. Changes the output from:
	```
	Building project images...
	........Project images built in 8s.

    Starting Mutagen sync process...
    ........Mutagen sync flush completed in 8s.
	```
	to:
	```
	Building project images...........
	Project images built in 8s.

    Starting Mutagen sync process...........
    Mutagen sync flush completed in 8s.
	```

3. We use short names here:
	```
	Waiting for containers to become ready: [web db]
	```
	but not here:
	```
	Waiting 120s for additional project containers [ddev-d11-redis ddev-d11-varnish] to become ready...
	```
	Change it to:
	```
	Waiting 120s for additional project containers [redis varnish] to become ready...
	```

## Manual Testing Instructions

Run `ddev debug rebuild`

Or

```
ddev config --webimage-extra-packages=vim
ddev add-on get ddev/ddev-redis
ddev start
...
Building project images.............
Project images built in 11s.
...
Waiting for containers to become ready: [web db]
...
Waiting 120s for additional project containers [redis] to become ready...
...
```


## Automated Testing Overview

<!-- Please describe the tests introduced by this PR, or explain why no tests are needed. -->

## Release/Deployment Notes

<!-- Does this affect anything else or have ramifications for other code? Does anything have to be done on deployment? -->

rpkoller reviewed (approved):
On a manual test the info is back to the previous state 
```
Building project images... 
.Project images built in 1s.
```
and according to @stasadev if the process to build takes longer leads to for example  `.... Project images built in 12s` ... looks good to me thanks you

rfay reviewed (commented):
I'm not sure, but it still looks like there's an errant newline:

```
Building project images...
........Project images built in 8s.
```

```
Building project images...
.Project images built in 1s.
```

stasadev commented:
```
Building project images...
........Project images built in 8s.
^
|
these dots are written to stderr
```

I can change it to this:

```
Building project images...
........
Project images built in 8s.
```

But it will look strange with just one dot:
```
Building project images...
.
Project images built in 1s.
```

---

And changing it to this variant is not trivial:

```
Building project images...........
Project images built in 8s.
```

Because if I don't add a new line for this output "Building project images...", then debug and verbose output will be printed to the same line:

```
Building project images...2025-01-17T15:59:52.996 Executing docker-compose -f /home/stas/ddev/d11/.ddev/.ddev-docker-compose-full.yaml build --progress=plain 
.
Project images built in 2s.
```

---

I leave everything as it is.

stasadev commented:
I changed the output to:

```
Building project images...........
Project images built in 8s.
```

And we used short names here:
```
Waiting for containers to become ready: [web db]
```
But not here:
```
Waiting 120s for additional project containers [ddev-d11-redis ddev-d11-varnish] to become ready...
```
I changed it to:
```
Waiting 120s for additional project containers [redis varnish] to become ready...
```

rfay commented:
I'm sorry to be a pain but I sure like this now.

```
Building project images....................................................................
Project images built in 1m6s.
 Network ddev-d11_default  Created
 Container ddev-d11-web  Created
 Container ddev-d11-db  Created
 Container ddev-d11-db  Started
 Container ddev-d11-web  Started
Starting Mutagen sync process...
.Mutagen sync flush completed in 3s.
For details on sync status 'ddev mutagen st d11 -l'
Waiting for containers to become ready: [web db]
```

rfay reviewed (approved):
Thank you!

rpkoller reviewed (approved):
Already the previous iteration looked ok for me and was an improvement. But after testing the latest iteration i have to agree that randy was right with his request, and it looks better and even more cleaned up now. thanks!

stasadev commented:
Update: Did the same with the mutagen output.

rfay commented:
Mutagen output looks great too, thank you!
